addons/im_livechat/demo/im_livechat_channel/im_livechat_session_14.xml

```diff
@@ -0,0 +1,72 @@
+            <field name="partner_id" ref="base.partner_admin"/>
+            <field name="channel_id" ref="im_livechat.livechat_channel_session_14"/>
+            <field name="unpin_dt" eval="DateTime.today()"/>
+            <field name="last_interest_dt" eval="DateTime.today()"/>
+            <field name="livechat_member_type">agent</field>
+        </record>
+        <record id="im_livechat.livechat_channel_session_14_guest" model="mail.guest">
+            <field name="name">Visitor</field>
+        </record>
+        <record id="im_livechat.livechat_channel_session_14_member_guest" model="discuss.channel.member">
+            <field name="guest_id" ref="im_livechat.livechat_channel_session_14_guest"/>
+            <field name="channel_id" ref="im_livechat.livechat_channel_session_14"/>
+            <field name="livechat_member_type">visitor</field>
+        </record>
+
+        <record id="livechat_channel_session_14_message_1" model="mail.message">
+            <field name="author_guest_id" ref="im_livechat.livechat_channel_session_14_guest"/>
+            <field name="author_id"/>
+            <field name="email_from">Visitor</field>
+            <field name="record_name">Visitor</field>
+            <field name="date" eval="datetime.now()"/>
+            <field name="create_date" eval="datetime.now()"/>
+            <field name="write_date" eval="datetime.now()"/>
+            <field name="body">Hi, could I get some help with the Discuss application?</field>
+            <field name="message_type">comment</field>
+            <field name="subtype_id" ref="mail.mt_comment"/>
+            <field name="res_id" ref="im_livechat.livechat_channel_session_14"/>
+            <field name="model">discuss.channel</field>
+        </record>
+        <record id="livechat_channel_session_14_message_2" model="mail.message">
+            <field name="author_id" ref="base.partner_admin"/>
+            <field name="date" eval="datetime.now() + timedelta(seconds=15)"/>
+            <field name="create_date" eval="datetime.now() + timedelta(seconds=15)"/>
+            <field name="write_date" eval="datetime.now() + timedelta(seconds=15)"/>
+            <field name="body">Sure, I'll add in the Discuss expert for you.</field>
+            <field name="message_type">comment</field>
+            <field name="subtype_id" ref="mail.mt_comment"/>
+            <field name="res_id" ref="im_livechat.livechat_channel_session_14"/>
+            <field name="model">discuss.channel</field>
+        </record>
+        <function model="discuss.channel" name="_add_members">
+            <value eval="[ref('im_livechat.livechat_channel_session_14')]"/>
+            <value name="partners" model="res.partner" eval="obj().env['res.partner'].browse(ref('base.partner_demo'))"/>
+            <value name="inviting_partner" model="res.partner" eval="obj().env['res.partner'].browse(ref('base.partner_admin'))"/>
+        </function>
```

Comment:
create date of the resulting message will be now(), which is actually before the previous message (livechat_channel_session_14_message_2). For better accuracy, we should probably:
- create the channel 1 minute ago
- post first messages
- add members
- post last messages